Dockerfile

```
@@ -1,10 +1,10 @@
 # keep variables beyond the single build stages, see https://stackoverflow.com/a/53682110/12529534
-ARG doguctl_version=0.12.2
+ARG doguctl_version=0.13.1
 
-FROM debian:12.7-slim as doguctlBinaryVerifier
+FROM debian:12.7-slim AS doguctlbinaryverifier
 ARG doguctl_version
 
-ENV DOGUCTL_SHA256=ac4b56f17a8b86ae398b45f33ba0c11bf4e2b80030d915d5b89207582f3ff648
+ENV DOGUCTL_SHA256=f82f17c6aa64f8d7ac1cc922043823660eb595a2ad45a42b47d10cf86696a85b
 ENV DOGUCTL_VERSION=$doguctl_version
 RUN mkdir packages
 COPY resources/usr/bin/doguctl-$DOGUCTL_VERSION.tar.gz /packages
```

```suggestion
COPY packages/doguctl-$DOGUCTL_VERSION.tar.gz /packages
```

And maybe we need to restructure the Dockerfile, because Docker always uses the cache when building this stage, even if the checksum or the doguctl-file is changed.
So the verification does not work.

---

Ugh, this is indeed a creepy feature. The verification should work every time as you said. T_T

---

The problem here is the unreferenced stage `doguctlbinaryverify`. `docker buildx` will optimize unreferenced stages away. I am going to fix this by calling the stage manually within `make build`